engine/runtime/src/test/scala/org/enso/compiler/test/pass/analyse/AliasAnalysisTest.scala

```diff
@@ -1219,8 +1219,7 @@ class AliasAnalysisTest extends CompilerTest {
     }
   }
 
-  "Alias analysis on typeset literals" ignore {
-    // FIXME: Not supported by new parser--needs triage (#5894).
+  "Alias analysis on typeset literals" should {
```

Comment:
Since #3878 was implemented the libraries team talks about the need of _"hash literals"_. E.g. the
```haskell
main = { x := 1, b := 2 }
```
syntax may need to be (properly) supported again in the future. CCing @jdunkerley to link with proper _"hash literals"_ request.

Reply:
Ok, tracked here: #6162